tasks/code_sync_status.rb

```diff
@@ -0,0 +1,93 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+
+# Puppet Task Name: code_sync_status
+require 'net/https'
+require 'uri'
+require 'json'
+require 'puppet'
+
+# CodeSyncStatus task class
+class CodeSyncStatus
+  def initialize(params)
+    @params = params
+  end
+
+  def execute!
+    puts syncstatus.to_json
```

Comment:
do we have to print (`puts`) the JSON on the screen? or just a regular `return` could work?

Reply:
Yeah, we do need to write to stdout, because we aren't (yet) using the Bolt ruby_task_helper. As a fairly raw task, something needs to handle performing the output so that Bolt can read and return it, and for now it's just this `puts`.

Reply:
I'll test that got to admit i copied and pasted threre

Reply:
Thanks for confirming @reidmv